Curs/src/views/common/common.js

```diff
@@ -24,17 +24,17 @@ interviewApp.NAV = function (options={}) {
             <nav class="col-md-12 navigation-bar">
                 <div class="col-md-4">
                     <!--Softvision logo-->
-                    <a href="new-evaluation.html"><img class="logo-image" src="src/views/img/logo-v2.png"/> </a>
+                    <a href="new-evaluation.html"><img id="logo-image-section" class="logo-image" src="src/views/img/logo-v2.png"/> </a>
                 </div>
                 <div class="col-md-4">
                     <!--Evaluation navbar button-->
-                    <a id="evaluation-page" class="border-bottom" href="#">Evaluations</a>
+                    <a id="evaluation-page" class="border-bottom" href="">Evaluations</a>
```

Comment:
Hmmm... any particular reason to remove the `#` character from the anchor's `href` attribute?

Reply:
I removed it because when I was trying to login that "#" was displayed in the link and I didn't liked it.
Probably in the next projects I will have routes for the links and then I will be able to put something in the href arrtibute. :)

Reply:
Sounds like an `event.preventDefault()` on the nav links click handlers might have done the job. It is usually good practice to have `href`s for your anchors, hence my question above.